lib/bucky/test_equipment/verifications/status_checker.rb

```diff
@@ -109,7 +109,8 @@ def make_target_links(args)
           doc = Nokogiri::HTML.parse(entity)
           links = []
           doc.xpath('//a').each do |node|
-            href = node.attr('href').split(' ')[0] # Patch for nokogiri's bug
+            href = node.attr('href')
+            href = href.split(' ')[0] unless href.nil? # Patch for nokogiri's bug
```

Comment:
How about using safe navigation operator.

```suggestion
            href = node.attr('href')&.split(' ')&.first
```

Reply:
FYI
```
irb(main):195:0* str1="a,b,c"
=> "a,b,c"
irb(main):196:0> str2=nil
=> nil
irb(main):197:0> p str1&.split(',')&.first
"a"
=> "a"
irb(main):198:0> p str2&.split(',')&.first
nil
=> nil
```

Reply:
COol! fixed